hub/api/dataset.py

```diff
@@ -144,9 +146,13 @@ def create_tensor(
                 For example, `htype="image"` would have `dtype` default to `uint8`.
                 These defaults can be overridden by explicitly passing any of the other parameters to this function.
                 May also modify the defaults for other parameters.
-            chunk_size (int): Optionally override this tensor's `chunk_size`. In short, `chunk_size` determines the size of files (chunks) being created to represent this tensor's samples.
+            chunk_size (int): Optionally override this tensor's `chunk_size`. In short, `chunk_size` determines the
+                size of files (chunks) being created to represent this tensor's samples.
                 For more on chunking, check out `hub.core.chunk_engine.chunker`.
             dtype (str): Optionally override this tensor's `dtype`. All subsequent samples are required to have this `dtype`.
+            default_compression (str): Optionally override this tensor's `default_compression`.
```

Comment:
This should just be called `compression`.

Reply:
i requested to change to `default_compression` to signify to users that this compression isn't the only one possible. since we are allowing every sample to have different compresssions

Reply:
thoughts?